build-and-stage.ps1

```diff
@@ -134,20 +164,38 @@ $tripletCount = $triplets.Length
 $tripletNum = 1
 foreach ($triplet in $triplets) {
    Write-Host "Installing for triplet $tripletNum/$tripletCount`: $triplet..."
-   Install-From-VcPkg -Package $PackageAndFeatures -Triplet $triplet -vcpkgExe $vcpkgExe
-   Exit-If-Error $LASTEXITCODE
+   Install-FromVcPkg -Package $PackageAndFeatures -Triplet $triplet -vcpkgExe $vcpkgExe
+   Exit-IfError $LASTEXITCODE
    Write-Host ""
    $tripletNum++
 }
+[Console]::Out.Flush()
 
 if($IsOnMacOS) {
    Write-Host ""
    Write-Host "============================================================"
    Write-Host "Creating universal binary"
    Write-Host "============================================================"
-   Write-Host "Running: makeUniversalBinary.sh..."
-   bash ./makeUniversalBinary.sh "vcpkg/installed/$($triplets[0])" "vcpkg/installed/$($triplets[1])" "$preStagePath"
+   $arm64Dir = "vcpkg/installed/$($triplets[0])"
+   $x64Dir = "vcpkg/installed/$($triplets[1])"
+   Write-Host "$arm64Dir, $x64Dir ==> $preStagePath"
+   ConvertTo-UniversalBinaries -arm64Dir "$arm64Dir" -x64Dir "$x64Dir" -universalDir "$preStagePath"
+   [Console]::Out.Flush()
+}
+
+Write-Host ""
+Write-Host "============================================================"
+Write-Host "Post-build step"
+Write-Host "============================================================"
```

Comment:
These output banners take up a lot of space. Maybe we could add a utility function, `Write-HostBanner` 

Maybe it's because I'm not great at powershell, but it is hard to tell what is actually happening at a high-level between all of the Write-Host statements.

Reply:
Fixed: 6e305d93edc6a8841d455d02013b239ed45d96c0